--- a/spit/CMakeLists.txt
+++ b/spit/CMakeLists.txt
@@ -47,11 +47,6 @@
 add_executable(pair pair.c )
 target_link_libraries(pair spitlib m )
 
-add_executable(stush stush.c)
-target_link_libraries(stush spitlib m pthread aio numa readline)
-
-set_target_properties(stush PROPERTIES EXCLUDE_FROM_ALL TRUE)
-
 add_executable(stat stat.c )
 target_link_libraries(stat spitlib m )
 
@@ -126,11 +121,9 @@
 target_link_libraries(testDiskStats spitlib m numa pthread)
 
 
-INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/spit.1 DESTINATION ${CMAKE_INSTALL_PREFIX}/man/man1)
-
-install(TARGETS spit fsfiller spitchecker raidcorrupt bdinfo dtest hist  DESTINATION bin)
+INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/spit.1 DESTINATION ${CMAKE_INSTALL_PREFIX}/share/man/man1)
 
-install(TARGETS stush PERMISSIONS SETUID WORLD_EXECUTE DESTINATION bin )
+install(TARGETS spit DESTINATION bin)
 
 # first check the position validation/collision collapsing is working
 #add_test(tds testDiskStats)
